ci/Jenkinsfile.macos

```diff
@@ -128,14 +131,14 @@ pipeline {
 
     stage('status-go') {
       steps {
-        sh "echo USE_NWAKU is ${USE_NWAKU}"
+        sh 'echo USE_NWAKU is ${USE_NWAKU}'
```

Comment:
What is the point of this? Parameter values are already easily accessible in Jenkins jobs.